app/views/shared/header/_locale_selector.html.erb

```diff
@@ -0,0 +1,8 @@
+<select id="locale-selector" onchange="window.open(this.value,'_self');">
+  <option value="#"><%= current_locale %></option>
+  <% ["en", "es", "fr", "it"].each do |locale| %>
+    <% if locale != current_locale %>
+      <option value="<%= locale %>"><%= locale %></option>
+    <% end %>
+  <% end %>
+</select>
```

Comment:
Would like to find a better way than `onchange="window.open(this.value,'_self');"` to redirect to the new localized url. Would like to have a ruby helper function triggered by the `onchange` that redirected to the current url with the new `:locale` param merged in to overwrite the existing `:locale` param in `params`. Didn't quite get there but was fine with this interim solution while I keep figuring out how to do that.